test/message/throw_custom_error.out

```diff
@@ -1,4 +1,4 @@
 *test*message*throw_custom_error.js:*
 throw ({ name: 'MyCustomError', message: 'This is a custom message' });
 ^
-MyCustomError: This is a custom message
```

Comment:
interestingly, there is sort of a practice of using Error.prototype.toString on object uncaught exceptions among the various js runtimes. this change could break eshost, among other things

Reply:
Can you elaborate? In what way are these errors caught at this stage?

Reply:
I really fail to see how this would have an impact. How do they handle existing errors that changed the `stack` property`? And it's also still possible to use `Error.prototype.toString` (while I do not know how they intercept the error at that point).